app/helpers/react_helper.rb

```diff
@@ -70,6 +71,12 @@ def related_sites(connections, query)
     end
   end
 
+  def external_tracking_code(affiliate)
+    return if affiliate.external_tracking_code.blank?
```

Comment:
Can we just return `affiliate.external_tracking_code` in here? I don't think there is a need for the guard clause because we get the same result without it